.github/workflows/ci.yml

```
@@ -178,6 +165,39 @@ jobs:
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up JDK 21 (Azul Zulu)
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'zulu'
+
+      - name: Run Kover Verification (Ubuntu only, excluding nodeApp)
+        if: startsWith(matrix.os, 'ubuntu')
```

actionlint (1.7.10)

[error] property "os" is not defined in object type {}

(expression)